libs/barista-components/stacked-series-chart/src/stacked-series-chart-bar.scss

```
@@ -0,0 +1,61 @@
+@import 'stacked-series-chart-shared';
```

please update to a relative path as those could be imported from a different location through the `includeStylePaths`.

```suggestion
@import './stacked-series-chart-shared';
```